### DISCUSSION TITLE
Como habilitar o apontamento de horas na Issue?

### DISCUSSION THREAD

**Discussion by ReinaldoCampos** [frontendbr/forum#2001] [Perguntas e Respostas]
Olá
Gostaria de saber como configurar a funcionalidade de apontamento de horas na Issue. Somente estava habilitando para o proprietário do repositório, no entanto não está mais sendo exibida. Veja imagem.

 
![image](https://user-images.githubusercontent.com/74680480/106360674-89e79880-62f8-11eb-809c-1c6cd2a6851a.png)

**Comment by wilsonneto-swe** [comment]
Olpá @ReinaldoCampos, o github não possui apontamento de horas "built-in", pelos botões que aparecem serem fora da paleta de cores do github, acredito se tratar de uma extensão de navegador.

Há várias que fazem este serviço, como por exemplo: https://everhour.com/integrations/github.

Mas até onde sei, o github não faz trackeamento de horas. A apenas meus R$ 0,50 de opinião, se quer fazer trackeamento de horas pessoais, use o Toggl Track, é fenomenal, se quiser fazer trackeamente de horas de uma equipe, minha opinião é: não faça... rs 

Forte abraço e sucesso ai!
